page-for-post-type.php

```diff
@@ -126,6 +129,20 @@ public function action_customize_register( WP_Customize_Manager $wp_customize )
 
 	}
 
+	public function filter_display_post_states( $post_states, $post ) {
```

Comment:
@timiwahalahti sorry can you add a docblock to this method to explain what it's doing and what the params are?